gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/sampling/statistics/ConnectedComponentsDistribution.java

```diff
@@ -30,7 +30,8 @@
  * Writes a mapping to the graph head, containing the component id and the number of graph elements
  * (vertices and edges) associated with it.
  */
```

Comment:
Not yours, I know, but would you please change it from "Writes a mapping to the graph head, containing ..." to "Returns a {@code Tuple3<String, Long, Long>}, containing ..."